webclient/src/components/elements/ProjectDialog.vue

```diff
@@ -2,10 +2,11 @@
   <div class='card'>
     <h4>{{ title }}</h4>
     <div class='form'>
-      <div class='input'>
-        <span class='input-label'> Project Name *</span>
-        <input type='text' placeholder='Project Name' v-model='project.Name'>
-      </div>
+      <RequiredTextBox 
```

Comment:
Can you replace the inputs in "create account" and "login" with these as well.

Reply:
@darwinfroese Will take this up tomorrow.

Reply:
Apologies for the delay. I am working on this. 
I have a question though. I am assuming, all these validation messages should be displayed on page load itself and hence I am using v-validate.initial. I hope this is correct.